synapse/models/crypto.py

```diff
@@ -13,7 +13,26 @@
 class CryptoModule(s_module.CoreModule):
     def getModelDefs(self):
         modl = {
+
             'types': (
+
+                ('crypto:currency:coin', ('str', {'lower': True}), {
+                    'doc': 'An individual crypto currency type.',
```

Comment:
should probably be explicit where the abbrevation comes from, e..g. coinbase.

Reply:
This name would be by convention only, we dont want to enforce a specific convention because it could preclude modeling coins that a specific source of aliases is not aware of.  We could *recommend* the use of coinbase names.